octopusdeploy/schema_deployment_action_slug_package_test.go

```diff
@@ -0,0 +1,141 @@
+package octopusdeploy
+
+import (
+	"testing"
+
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/packages"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExpandDeploymentActionSlugPackages(t *testing.T) {
+	actual := expandDeploymentActionSlugPackages(nil)
+	require.Nil(t, actual)
+
+	actual = expandDeploymentActionSlugPackages([]interface{}{})
+	expected := []packages.DeploymentActionSlugPackage{}
+	require.Equal(t, expected, actual)
```

Comment:
No real reason besides it is convenient and I copied that pattern from another unit test 😄. It would be better to split into individual well named tests - which I'll do

Reply:
Thank you :bow: